bootstrap.js

```diff
@@ -24,7 +24,7 @@ const runApplication = async () => {
     await startServer()
   } catch (error) {
     console.error('Fatal exception on startup!');
-    console.error(`${err}`);
+    console.error(`${error}`);
     process.exit(-1);
```

Comment:
Why don't we use logger?

Reply:
Yes, I definitely want to use a logger library! I've just decided no to do it from scratch, since I was trying to solve a very specific problem.
I will create an issue, so we can implement it!